postman/Get Broadcast Open Clicks.postman_collection.json

```diff
@@ -0,0 +1,336 @@
+{
+	"info": {
+		"_postman_id": "886568e4-566c-42d1-833e-58b3de8c0833",
+		"name": "Get Broadcast Open Clicks",
+		"description": "This collection will return a list of link in a randomly selected broadcast",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Get Account",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "5d0f8297-d924-4b24-b51f-f2c4af5ffd5f",
+						"exec": [
+							"pm.test(\"Check consumer_key is defined and not empty\", function(){",
+							"   pm.expect(pm.environment.has(\"consumer_key\")).to.equal(true) && pm.expect(pm.environment.get(\"consumer_key\")).not.eql('');",
+							"});",
+							"",
+							"pm.test(\"Check consumer_secret is defined and not empty\", function(){",
+							"   pm.expect(pm.environment.has(\"consumer_secret\")).to.eql(true) && pm.expect(pm.environment.get(\"consumer_secret\")).not.eql('');",
+							"});",
+							"",
+							"pm.test(\"Check access_token is defined and not empty\", function(){",
+							"   pm.expect(pm.environment.has(\"access_token\")).to.eql(true) && pm.expect(pm.environment.get(\"access_token\")).not.eql('');",
+							"});",
+							"",
+							"pm.test(\"Check token_secret is defined and not empty\", function(){",
+							"   pm.expect(pm.environment.has(\"token_secret\")).to.eql(true) && pm.expect(pm.environment.get(\"token_secret\")).not.eql('');",
+							"});",
+							"",
+							"pm.test(\"Successfully get an account Id\", function(){",
+							"    pm.response.to.have.status(200);",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.entries.length).to.be.above(0);",
+							"    pm.globals.set(\"accountId\",jsonData.entries[0].id ) ;   ",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Lists on Account",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "4d22c8fa-8695-4098-a275-10658e8aaea5",
+						"exec": [
+							"var jsonData = pm.response.json();",
+							"",
+							"pm.test(\"Retrieved more one or more lists for account\", function(){",
+							"    pm.expect(jsonData.total_size).is.be.above(0);",
+							"});",
+							"",
+							"pm.test(\"Successfully selected a random listId\", function() {",
+							"    var listId = Math.floor((Math.random() * jsonData.total_size ));",
+							"    if (typeof jsonData.entries[0].id == 'number' ) {",
+							"        pm.globals.set(\"listId\",jsonData.entries[0].id );",
```

Comment:
The previous 2 and next 1 lines can be deleted. This always picks the first list, the previous test already checked that a list exists, and id is always a number.

Reply:
Removed redundant asserts